.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Required Information**
+Toolchain (if is a build problem):
+```
+$ cargo --version
+$ rustc --version
+$ nasm --version # if on x86_64
+```
+Version:
+```
+$ rav1e --version
+```
+Operating system:
+```
+$ uname -a
+...
```

Comment:
```suggestion
** Console Output **
Please paste the console output below if it is relevant, 
<Details>
Snippet here 
</Details>

```


We could also add one more line saying 
`Console Output` where we could ask people to paste some debug information if relevant.

Reply:
our console output is not that interesting, but sure